web/js/init.js

```diff
@@ -460,6 +460,13 @@ $(document).ready(() => {
     placement: 'bottom',
     container: 'body',
   });
+
+  $('.download-ics-btn').tooltip({
+    title: 'Download ICS File',
```

Comment:
People probably don't know what an ICS file is, so it might be better to say "Download as Calendar" or something.